aws_doc_sdk_examples_tools/config/services.yaml

```diff
@@ -2424,6 +2424,22 @@ memorydb:
   tags:
     product_categories: {}
   version: memorydb-2021-01-01
+neptune:
+  long: '&neptunelong;'
+  short: '&neptune;'
+  sort: Forecast
+  sdk_id: forecast
+  expanded:
+    long: Amazon Neptune Service
+    short: Neptune
+  blurb: is a fully managed graph database service by AWS, designed specifically for handling complex relationships and connected datasets at scale.
+  guide:
+    subtitle: User Guide
+    url: neptune/latest/userguide/intro.html
+  api_ref: neptune/latest/userguide/api.html
```

Comment:
This should typically be the actual API ref guide, not an internal user guide page, unless there's a good reason for it.